### PR TITLE
@dzucconi -> Filter partners

### PR DESCRIPTION
**Pull request by 1aurabrown**
results in this
screen shot 2016-01-22 at 3 32 47 pm

lmk if you think it should be organized differently

### CODE REVIEW

File: schema/aggregations/filter_partners_aggregation.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    },
+    aggregations: {
+      type: new GraphQLList(PartnersAggregationResultsType),
+      resolve: (obj) =>
+        Object.keys(obj.aggregations).map(function (key) {
+          const aggregation = {};
+          aggregation.counts = obj.aggregations[key];
+          aggregation.slice = key;
+          return aggregation;
+        }),
```
Comment (dzucconi) [inline schema/aggregations/filter_partners_aggregation.js]: I'm sorta surprised this is valid syntax?

**Comment by mzikherman** [comment]
I like this. I couldn't find anything to nit pick over (but I'm no @dzucconi ). Looks really nice!

File: schema/aggregations/filter_partners_aggregation.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+export const PartnersAggregationResultsType = new GraphQLObjectType({
+  name: 'PartnersAggregationResults',
+  description: 'The results for one of the requested aggregations',
+  fields: () => ({
+    slice: {
+      type: PartnersAggregation,
+    },
+    counts: {
+      type: new GraphQLList(AggregationCountType),
+      resolve: (obj) => parseCounts(obj),
```
Comment (dzucconi) [inline schema/aggregations/filter_partners_aggregation.js]: Could just be `resolve: parseCounts`

**Comment by dzucconi** [comment]
Looks good to me! Nice work

File: schema/aggregations/filter_partners_aggregation.js (hunk 1 above)
Comment (dzucconi) [inline schema/aggregations/filter_partners_aggregation.js]: I think this can be done in a single `reduce`

**Pull request merged**